books/free-programming-books-ar.md

```diff
@@ -38,6 +40,12 @@
 * [سبيلك المختصر لتعلم لغة c# - برمجة الواجهات](https://www.mobarmijoun.com/2014/04/c_19.html) - Khaled Al-Saadani (Arab Coders Academy)
 
 
+### Embedded Systems
+
+* [ متحكمات AVR](https://simplyarduino.com/%d9%83%d8%aa%d8%a7%d8%a8-%d9%85%d8%aa%d8%ad%d9%83%d9%85%d8%a7%d8%aa-avr-%d8%a3-%d8%af-%d9%85%d8%ad%d9%85%d8%af-%d8%a5%d8%a8%d8%b1%d8%a7%d9%87%d9%8a%d9%85-%d8%a7%d9%84%d8%b9%d8%af%d9%88%d9%8a/)
+* [Simply AVR](http://simplyarduino.com/%D9%83%D8%AA%D8%A7%D8%A8-simply-avr/)
```

Comment:
couldn't download this

Reply:
Downloadable targets points to storage sites.

![image](https://user-images.githubusercontent.com/3125580/135683158-7c5151c9-0707-4c7c-a96f-85a7e0da4a95.png)

Should we accept this attending to  [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) guidelines?

>- we don't accept files hosted on google drive, dropbox, mega, scribd, issuu and other similar file upload platforms
>- use the link with the most authoritative source (meaning the author's website is better than the editor's website, which is better than a third party website)

Reply:
that's ok, but one of those links is dead and the other 2 point to zip files, which seems bad.